### PR TITLE
feat(alpha): add optional timeseriesExternalId type to monitoring task

### PR DESCRIPTION
**Pull request by saagar-cognite**

### CODE REVIEW

**Comment by polomani** [comment]
not chore, it's `feat(alpha):`

**Comment by saagar-cognite** [comment]
@polomani just the PR title or do I have to make a new commit as well ?

**Comment by polomani** [comment]
@saagar-cognite it's the commit message that is supposed to be when merged to master. (most important to check when you actually merge it)

**Comment by saagar-cognite** [comment]
@polomani makes sense. I will take care of that.
Is there anything left to get this approved ?

File: packages/alpha/src/types.ts [hunk 1]
```diff
@@ -56,6 +56,7 @@ export interface MonitoringTaskThresholdModel {
 export interface MonitoringTaskDoubleThresholdModel {
   externalId: 'double_threshold';
   timeseriesId: CogniteInternalId;
+  timeseriesExternalId?: CogniteExternalId;
```
Comment (polomani) [inline packages/alpha/src/types.ts:59]: should we also add it to `interface MonitoringTaskDoubleThresholdModelCreate`?

**Review by polomani — APPROVED** [review]

**Pull request merged**